<ui version="4.0" >
 <author>Petr Vanek &lt;petr@scribus.info></author>
 <comment>Base class for color wheel dialog</comment>
 <class>CWDialog</class>
 <widget class="QDialog" name="CWDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Color Wheel</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="ColorWheel" native="1" name="colorWheel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="toolTip" >
      <string>Click the wheel to get the base color. Its color model depends on the chosen tab.</string>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="1" column="1" >
    <widget class="QGroupBox" name="groupBox3" >
     <property name="title" >
      <string>Result Colors</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QTableWidget" name="currentColorTable" >
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="rowCount" >
         <number>3</number>
        </property>
        <property name="columnCount" >
         <number>5</number>
        </property>
        <row>
         <property name="text" >
          <string>CMYK</string>
         </property>
        </row>
        <row>
         <property name="text" >
          <string>RGB</string>
         </property>
        </row>
        <row>
         <property name="text" >
          <string>HSV</string>
         </property>
        </row>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="ColorListBox" native="1" name="colorList" >
        <property name="minimumSize" >
         <size>
          <width>240</width>
          <height>160</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Colors of your chosen color scheme.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox3_2" >
     <property name="title" >
      <string>Color Scheme Method</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="angleLabel" >
          <property name="maximumSize" >
           <size>
            <width>100</width>
            <height>32767</height>
           </size>
          </property>
          <property name="text" >
           <string>Angle:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="angleSpin" >
          <property name="toolTip" >
           <string>Difference between the selected value and the counted ones. Refer to documentation for more information.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" >
       <widget class="QComboBox" name="typeCombo" >
        <property name="maximumSize" >
         <size>
          <width>280</width>
          <height>32767</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Select one of the methods to create a color scheme. Refer to documentation for more information.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton" >
       <property name="toolTip" >
        <string>Merge created colors into the document colors</string>
       </property>
       <property name="text" >
        <string>&amp;Merge</string>
       </property>
       <property name="shortcut" >
        <string>Alt+M</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceButton" >
       <property name="toolTip" >
        <string>Replace created colors in the document colors</string>
       </property>
       <property name="text" >
        <string>&amp;Replace</string>
       </property>
       <property name="shortcut" >
        <string>Alt+R</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="toolTip" >
        <string>Leave colors untouched</string>
       </property>
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox2" >
     <property name="title" >
      <string>Preview:</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="0" >
       <widget class="QLabel" name="previewLabel" >
        <property name="minimumSize" >
         <size>
          <width>280</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>280</width>
          <height>100</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Sample color scheme.</string>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Box</enum>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QComboBox" name="defectCombo" >
        <property name="minimumSize" >
         <size>
          <width>280</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Simulate common vision defects here. Select type of the defect.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel1" >
        <property name="text" >
         <string>Vision Defect Type:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <spacer name="verticalSpacer" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QTabWidget" name="colorspaceTab" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabCMYK" >
      <attribute name="title" >
       <string>CMYK</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>11</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel4" >
           <property name="text" >
            <string>C:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="cSpin" >
           <property name="suffix" >
            <string> %</string>
           </property>
           <property name="maximum" >
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel5" >
           <property name="text" >
            <string>M:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mSpin" >
           <property name="suffix" >
            <string> %</string>
           </property>
           <property name="maximum" >
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel6" >
           <property name="text" >
            <string>Y:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="ySpin" >
           <property name="suffix" >
            <string> %</string>
           </property>
           <property name="maximum" >
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel7" >
           <property name="text" >
            <string>K:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="kSpin" >
           <property name="suffix" >
            <string> %</string>
           </property>
           <property name="maximum" >
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel1_2" >
           <property name="text" >
            <string>RGB:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="rgbLabel" >
           <property name="frameShape" >
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel1_2_2" >
           <property name="text" >
            <string>HSV:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="hsvLabel" >
           <property name="frameShape" >
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>254</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRGB" >
      <attribute name="title" >
       <string>RGB</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>11</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="textLabel8" >
             <property name="text" >
              <string>R:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="rSpin" >
             <property name="maximum" >
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="textLabel9" >
             <property name="text" >
              <string>G:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="gSpin" >
             <property name="maximum" >
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="textLabel10" >
             <property name="text" >
              <string>B:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="bSpin" >
             <property name="maximum" >
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="textLabel3" >
             <property name="text" >
              <string>CMYK:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="cmykLabel" >
             <property name="frameShape" >
              <enum>QFrame::Box</enum>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="textLabel1_2_2_2" >
             <property name="text" >
              <string>HSV:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="hsvLabel2" >
             <property name="frameShape" >
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Plain</enum>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabHSV" >
      <attribute name="title" >
       <string>HSV</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>11</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="textLabel4_2" >
             <property name="text" >
              <string>H:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="hSpin" >
             <property name="maximum" >
              <number>359</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="textLabel5_2" >
             <property name="text" >
              <string>S:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sSpin" >
             <property name="maximum" >
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="textLabel6_2" >
             <property name="text" >
              <string>V:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="vSpin" >
             <property name="maximum" >
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="textLabel1_2_3" >
             <property name="text" >
              <string>RGB:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="rgbLabel2" >
             <property name="frameShape" >
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Plain</enum>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="textLabel1_2_2_3" >
             <property name="text" >
              <string>CMYK:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="cmykLabel2" >
             <property name="frameShape" >
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Plain</enum>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDocument" >
      <attribute name="title" >
       <string>Document</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>11</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="ColorListBox" native="1" name="documentColorList" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>ColorWheel</class>
   <extends>QWidget</extends>
   <header>colorwheelwidget.h</header>
  </customwidget>
  <customwidget>
   <class>ColorListBox</class>
   <extends>QWidget</extends>
   <header>ui/colorlistbox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >colorwheelwidget.h</include>
  <include location="local" >ui/colorlistbox.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
